build.gradle

```diff
@@ -1,28 +1,35 @@
-apply plugin: 'checkstyle'
-apply plugin: 'application'
-apply plugin: 'eclipse'
-apply plugin: 'idea'
-apply plugin: 'maven-publish'
-
+plugins {
```

Comment:
The indentation in this file seems very weird. Try a `Reformat Code`?

Reply:
it's default indentation with fabric code style

Reply:
maybe github is displaying them weird but they don't all look like tabs